### DISCUSSION TITLE
OpenWebUI / ollama Error 500

### DISCUSSION THREAD

**Discussion by jsapede** [community-scripts/ProxmoxVE#26] [Q&A] [answered]
helle,
installed this morning this lxc including ollama but each time i try to make a test i get error 500 : 

![image](https://github.com/user-attachments/assets/6cf024f2-81d5-4218-b579-1a0692b30e9e)

tried to change ip in the .env config like here https://github.com/tteck/Proxmox/discussions/3994#discussioncomment-11075421 bit with no additionnal success

**Comment by jsapede** [comment]
moreover looks like in the new site there isnt anymore ollama install available ?

**Comment by MickLesk** [comment]
Can you open with your IP and the Port to Check ollama is working?

Can you share Output of systemctl Status?

@havardthom can you take a look?

**Comment by havardthom** [comment]
Check Ollama logs after trying the chat: `journalctl -u ollama.service -n` (in LXC console)

Do you have enough resources allocated to the LXC for the model you are trying? Could be out of memory error, check LXC resources while you try the chat

**Reply by jsapede** [reply]
tried with ollama separate install from exisiting script and seems to work 
will retry with ollama install integrated to script

**Comment by jsapede** [comment, marked as answer]
reinstalled from zero and seems to work !
is there a way to have intel acceleration on this install script ?

**Reply by havardthom** [reply]
Hardware acceleration for Intel iGPU is currently not supported in Ollama, it will be after this PR is merged: https://github.com/ollama/ollama/pull/5593

Hardware acceleration for Intel dGPU requires building Ollama from source code with intel-basekit and Intel drivers (this script covers it: https://github.com/community-scripts/ProxmoxVE/blob/main/install/ollama-install.sh)

Regardless, setting up Intel hardware acceleration currently takes too long for it to be user friendly, it would slow down the script install too much. Users that want it can set it up themselves until a more effective install method appears (hopefully Ollama will add support for it in their precompiled binary eventually)

**Comment by Solear** [comment]
I have the same error like the Threadstarter. 

> 500: Ollama: 500, message='Internal Server Error', url='http://0.0.0.0:11434/api/chat'

Reinstalled it new with 8 GB and 6 cores, no luck.

**Reply by MickLesk** [reply]
try 127.0.0.1

**Reply by Solear** [reply]
127.0.0.0 and real IP doesnt work.
But I changed the model from llama3.2 to llama3.1, and now it works...

**Reply by MickLesk** [reply]
ive the same issue a few weeks ago. Ive removed all :D

**Comment by Vendo233** [comment]
I have the same error
500: Ollama: 500, message='Internal Server Error', url='http://0.0.0.0:11434/api/chat'
what can I do?

**Comment by Vendo233** [comment]
I have tried to reinstall, tried localhost, 127.0.0.1 and nothing helped
